src/lib/components/Map/Map.js

```diff
@@ -255,7 +255,7 @@ class Map extends Component {
               this.map.moveLayer(sublayers[s]);
             }
           }
-          orderLayers(sortedLayers, map, nextLayerId);
+          sortLayers(map, sortedLayers, nextLayerId);
```

Comment:
@ciremusyoka since we have multiple versions of both `gisida` and `gisida-react` and our clients can use a mix of versions, we may need to keep both `orderLayers()` and `sortLayers()`. This is a perfect example of deprecation vs removal of old code.

If a client has a version of `gisida-react` which is importing `sortLayers` from a version of `gisida` which is not exporting `sortLayers`, this line (and `301`) should fall back to `orderLayers`. Otherwise, in this scenario we would get a TypeError that undefined is not a function.

```
typeof sortLayers === 'function' ? sortLayers(...) : orderLayers(...)
```

If we were to release this as v2.0.0 then we could do full removal of the deprecated code, but since this will be another 1.x.y release then let's go with the deprecation route.

Reply:
@cKellyDesign  should i also update the `orderLayer` to match `sortLayers`?

Reply:
@ciremusyoka in terms of ordering functionality, yes these should essentially result in the same ordering outcome (only that `sortLayers` handles heatmap layers as well. Otherwise we can simply add the heatmap support to `orderLayers` and remove `sortLayers`

Reply:
@cKellyDesign I think it would be better to update the `orderLayer` to match `sortLayer`  and update `map.js` as suggested [here](https://github.com/onaio/gisida-react/pull/175#discussion_r379044883) and depreciate it with time. This will help keep `gisida-react` as a view library.